api/src/database/migrations/20191112124521_initial.ts

```diff
@@ -15,6 +15,8 @@ export async function up(knex: Knex): Promise<any> {
         table.string('description', _MAX_DESCRIPTION_LENGTH).notNullable();
         table.string('description_link', _MAX_URL_LENGTH);
         table.string('logo_link', _MAX_URL_LENGTH);
+        table.string('mvp_link', _MAX_URL_LENGTH);
```

Comment:
I think you need these as a new migration, right? That way existing DBs get the columns added. Nothing is really in production yet though so maybe this is OK???

Reply:
We discussed this. I'm OK with it.